src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OdsController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Organisations.Api.ViewModels.Organisations;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Controllers
+{
+    //[Authorize(Policy = Policy.CanAccessOrganisation)]
+    [Route("api/v1/ods")]
+    [ApiController]
+    [Produces("application/json")]
+    public sealed class OdsController : Controller
+    {
+        [HttpGet]
+        [Route("{odsCode}")]
+        public ActionResult GetByOdsCodeAsync(string odsCode)
+        {
+            if (odsCode is null)
+                throw new ArgumentNullException(nameof(odsCode));
+
+            if (string.IsNullOrWhiteSpace(odsCode))
+                throw new ArgumentException($"{nameof(odsCode)} cannot be empty.", nameof(odsCode));
+
+            // Canned data
+            return Ok(new OrganisationViewModel
```

Comment:
Although the data we are returning is the same as the organisation view model I consider this endpoint as a separate query. Further down the line we might want to include additional information from the ODS endpoint but not effect the organisations endpoints, so I think it would be better to use a different "viewmodel" contract e.g OdsViewModel. This keeps the concerns separate.

Reply:
Good point, I agree.